packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -17,11 +17,6 @@ export type CdpEvent = keyof ProtocolMapping.Events
 
 const debugVerbose = debugModule('cypress-verbose:server:browsers:cdp_automation')
 
-export type CyCookie = Pick<chrome.cookies.Cookie, 'name' | 'value' | 'expirationDate' | 'hostOnly' | 'domain' | 'path' | 'secure' | 'httpOnly'> & {
-  // use `undefined` instead of `unspecified`
-  sameSite?: 'no_restriction' | 'lax' | 'strict'
```

Comment:
any value in preserving the comment here in the new type so we avoid running into [this](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/cookies.ts#L55), even though this is just the type?